### PR TITLE
1084 feature update categories and subcategories mapping using the latest information

### PR DESCRIPTION
**Pull request by bingzhang**
## Description
_Please provide a summary of the pull request and the issue it fixes. Please add necessary details, context, dependencies, explanation of when review is needed (see next section), etc._

**Fixes #(add issue number here and remove parentheses)**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [ ] Within a week
- [x] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

**Comment by sandeep-ps** [comment]
Hi, @bingzhang, Have you tested this by doing a calendar event download? If not, please do that to make sure that the event download works fine. Thanks.

**Comment by bingzhang** [comment]
> Hi, @bingzhang, Have you tested this by doing a calendar event download? If not, please do that to make sure that the event download works fine. Thanks.

the current webtool still use old mapping. so there are many unknown type now. Shall we skip the old mapping events? Or we can delay the merge of this pr.

File: utilities/constants.py [hunk 1]
```diff
@@ -89,6 +83,15 @@
         "Swim & Dive",
         "Wrestling",
         "Volleyball"
+    ],
+    "Recreation, Health & Fitness": [
```
Comment (sandeep-ps) [inline utilities/constants.py:87]: `Athletics` in line 66 will need to change to Big10 Athletics, right?

Comment (sandeep-ps) [reply utilities/constants.py:87]: It looks like I did not think about that change in the issue description. I have updated it now. Thanks.

Comment (bingzhang) [reply utilities/constants.py:87]: @sandeep-ps looks good now?

Comment (sandeep-ps) [reply utilities/constants.py:87]: Yes, thanks.

**Review by sandeep-ps — APPROVED** [review]

**Review by minump — APPROVED** [review]
Approving

**Pull request merged**